content/docs/concepts/asm/_index.md

```diff
@@ -0,0 +1,275 @@
+ 
+ <!--
+ - _SCALE_ = `18`, the decimal places in fixed-point integer representation.
+- _SCALE_FACTOR_ = `10 ** SCALE`, the multiplier for scaling numbers to the required scale.
+- _SECONDS_IN_YEAR_ = `365 days`, a year is assumed to have 365 days for interest rate calculations.   
+-->
+
+### Protocol primitives
+
+Essential primitives of ASM are: collateral, exchange rate price data, ACU, and the CDP.
+
+#### Collateral
+
+Auton borrowing is collateralized by depositing collateral token into a CDP. The amount and value of collateral backing a CDP is determined by collateralization and liquidation ratios set in the [ASM configuration](/concepts/asm/#asm-configuration). Failure by a [borrower](/concepts/asm/#stabilization-roles) to maintain these ratios results in a CDP becoming liquidatable. In a liquidation scenario a [liquidator](/concepts/asm/#stabilization-roles) is able to assume the debt position, repay outstanding debt, and receive the position's remaining collateral in return.
+
+Autonity's native protocol asset [Newton (NTN)](/concepts/protocol-assets/newton/) is used as the collateral token.
+
+{{% alert title="Info" %}}
+A future iteration of ASM will enable the use of [Liquid Newton (LNTN)](/concepts/protocol-assets/liquid-newton/) as collateral token.
+{{% /alert %}}
+
+#### Exchange rate price data
+ASM sources price data via Autonity's [oracle network](/concepts/oracle-network/), retrieving the data on-chain by contract interactions with the [Oracle Contract](/concepts/architecture/#autonity-oracle-contract).
+
+Oracle price data is used for two purposes:
+
+- for the ACU currency basket symbols to compute the ACU _value_
+- for CDP borrowing to compute the value of Collateral Token (NTN) in ATN and determine the _borrow limit_.
+
+Oracle price data is computed per the [Oracle protocol](/concepts/oracle-network/#oracle-protocol), updated periodically in [voting rounds](/glossary/#voting-round).
+
+#### ACU
+
+Auton price has the [Auton Currency Unit (ACU)](/glossary/#acu) as the Stabilization Target to which it _mean-reverts_. ACU is an index value computed from a basket of free-floating currencies. Use of a currency basket minimises exposure to an individual currency's FX exchange risk. The index value is computed from the basket, weighted _pro rata_ to each currency's share i.e. _quantity_ in the basket. Basket quantities are set at network genesis and may be modified by governance.
+
+ASM then functions to maintain Auton-to-ACU value, '_mean reverting_' to this value by the CDP stabilization mechanism. 
+
+ACU value is kept current by protocol recomputing the value at the end of each oracle voting round when price data for all the basket currencies is available. See the Protocol Only Reference functions [`modifyBasket()`](/reference/api/aut/op-prot/#modifybasket-acu-contract) [`update()`](/reference/api/aut/op-prot/#update-acu-contract) for more detail.
+
+#### CDP
```

Comment:
This CDP section here is missing crucial details about how these various quantities are computed, ie. formulas for borrow interest rate, borrow limit, how interest is computed, ...